powertools-idempotency/powertools-idempotency-core/src/main/java/software/amazon/lambda/powertools/idempotency/IdempotencyConfig.java

```diff
@@ -15,9 +15,10 @@
 package software.amazon.lambda.powertools.idempotency;
 
 import com.amazonaws.services.lambda.runtime.Context;
-import java.time.Duration;
 import software.amazon.lambda.powertools.idempotency.internal.cache.LRUCache;
 
+import java.time.Duration;
```

Comment:
I think you don't have [checkstyle](https://github.com/aws-powertools/powertools-lambda-java/blob/main/CONTRIBUTING.md#dev-setup) applied.

Reply:
It should be fixed now. I think it is because we only format uncommitted and if I forget to format before commit, I am missing some files..